spec/fixtures/.dotenv.dup

```diff
@@ -0,0 +1,3 @@
+FOO=first
+BAR=tender
+FOO=last
```

Comment:
@anddriex You many need to add this file in the cabal file [here](https://github.com/stackbuilders/dotenv-hs/blob/master/dotenv.cabal#L48-L50)

Reply:
Done.